automation/validate-pvc-name-stability.py

```diff
@@ -0,0 +1,105 @@
+#! python
+
+import logging
+import os
+import os.path
+import yaml
+import subprocess
+import sys
+
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
+
+
+def validatePVCNames(path, liveTemplates):
+    templates = [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+    errors = []
+    for templateFilename in templates:
+        with open(os.path.join(path, templateFilename), 'r') as stream:
+            try:
```

Comment:
Can you make this `try` block smaller, so it is only around statements that can throw exceptions? I think, it is probably only `template = yaml.safe_load(stream)`